frontend/src/store/playerElement.ts

```diff
@@ -154,6 +180,19 @@ class PlayerElementStore {
         (sub) => sub.srcIndex === playbackManager.currentSubtitleStreamIndex
       );
 
+      const attachedFonts =
+        playbackManager.currentMediaSource?.MediaAttachments?.filter((attach) =>
+          this._isSupportedFonts(attach.MimeType || '')
+        )
+          .map((attach): string | undefined => {
+            if (attach.DeliveryUrl && serverAddress) {
+              return `${serverAddress}${attach.DeliveryUrl}`;
+            }
+          })
+          .filter((attach) => attach !== undefined && attach !== null) as
+          | string[]
+          | undefined;
```

Comment:
```suggestion
          .filter((a): a is string => a !== undefined && a !== null) ?? [];
```

Use a type guard so types are correct after filtering